CHANGES.md

```diff
@@ -6,6 +6,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### What's new
 
+* Updated to Metronome 0.6.33 which has the following benefits: When querying run detail with embed=history, successfulFinishedRuns and failedFinishedRuns contains new field tasks which is an array of taskIds of that finished run. This will allow people to query task ids even for finished job runs.  Updated to the latest version of cron-utils 9.0.0 and removed threeten-backport. This fixes a number of cron related issues in the underlying dependencies.  Fixed a bug when task status was not updated after the task turned running (when querying embed=activeRuns).  Fixes DCOS_OSS-5166 where metronome did not use the revive operation
+
```

Comment:
Huhu Ken! We don't need this here. It will **not** be news to our 1.14.0 users.

I really do hope that https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines#major-releases explains the "why". Let me know if this makes sense to you.

Will remove this entry in an editorial round.